orchagent/aclorch.cpp

```diff
@@ -122,6 +122,8 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
 
     sai_attribute_value_t value;
 
+    value.aclfield.enable = true;
```

Comment:
this is not the correct approach. range needs to be created separately. and then mark the `aclfield.enable` = true. because sai_attribute_t is a union, it cannot be used as both range and aclfield at the same time.